### PR TITLE
Create LIT6511Sawasew.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record

### CODE REVIEW

File: new/LIT6511Sawasew.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6511Sawasew" passive="BNFabb147_a2"/>
+                </listRelation>
+
```
Comment (eu-genia) [inline new/LIT6511Sawasew.xml:58]: close the bibliography div here and open an edition div
```
</div>
<div type="edition">
```

File: new/LIT6511Sawasew.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                <note>The incipit is taken from <ref corresp="BNFabb147" type="mss"></ref></note>
+                <div type="textpart" subtype="incipit">
+                  <ab>
```
Comment (eu-genia) [inline new/LIT6511Sawasew.xml:61]: ab xml:lang="gez"

File: new/LIT6511Sawasew.xml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    ይህ፡ መጽሐፍ፡ ለደቀ፡ መዝሙር፡ ከመምህር፡ ይሻላቸዋል፡ የሰዋስውም፡ ፋናው፡ ነው፡ አስተዋይ፡ ሰው፡ ቢመለከተው። ናሁ፡ እጽሕፍ፡
+                    አነ፡ ነገረ፡ አገባብ፡ ዘኵሎን፡ ኆኅያት፡ እምነ፡ ሀ፡ እስከ፡ ፈ። ፍኖተ፡ ጼሐ፡ ፍኖተ፡ ተመርሐ፡ እምፍኖት፡ ነስሐ፡ እደዊሁ፡
+                    ሰፍሐ፡ እደዊሁ፡ ተሞቅሐ፡ ሰይፈ፡ መልሐ፡ ማኅተመ፡ ፈትሐ፡ ቀርነ፡ ነፍሐ፡ ሞጣሕተ፡ ተሞትሐ፡ ክርዳደ፡ አስተናጽሐ፡ ደመ፡ ነዝኃ፡ ቀደሰ፡ ርሱሐ፡ አዕረፈ፡ ሥሩሐ፡
+                    ነገፈ፡ ላሀ፡ ሐነጸ፡ ጽርሐ፡ ሀብለ፡ አንኃ፡ በቃል፡ ጸርሐ፡
+                  </ab>
+                </div>
+                  <div type="textpart" subtype="explicit">
+                    <ab>
```
Comment (eu-genia) [inline new/LIT6511Sawasew.xml:69]: idem

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6511Sawasew.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT6511Sawasew.xml:58]: ok

File: new/LIT6511Sawasew.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6511Sawasew.xml:63]: here both amharic indroductory formula is used, the body is in Geez.

Comment (eu-genia) [reply new/LIT6511Sawasew.xml:63]: then mark up accordingly

Comment (eu-genia) [reply new/LIT6511Sawasew.xml:63]: so you need for the text
`<textLang mainLang="gez" otherLangs="am"></textLang>
`and in the incipit the amharic part marked as `<foreign xml:lang="am">`

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**